src/EventListener/UserNavigationListener.php

```diff
@@ -26,7 +26,7 @@ public function __construct(
         private readonly Connection $connection,
         private readonly RequestStack $requestStack,
         private readonly ScopeMatcher $scopeMatcher,
-        private readonly Security $security,
+        private readonly AuthorizationCheckerInterface $security,
```

Comment:
Please also change the variable name and its usage in the code.

```suggestion
        private readonly AuthorizationCheckerInterface $authorizationChecker,
```